log/log.go

```diff
@@ -302,3 +303,15 @@ func Fatalln(args ...interface{}) {
 func Fatalf(format string, args ...interface{}) {
 	baseLogger.sourced().Fatalf(format, args...)
 }
+
+type errorLogWriter struct{}
+
+func (errorLogWriter) Write(b []byte) (int, error) {
+	baseLogger.sourced().Error(string(b))
+	return len(b), nil
+}
+
+// NewErrorLogger return log.Logger to be used as ErrorLog to catch HTTP server errors.
```

Comment:
"return" -> "returns a"

How about: "NewErrorLogger returns a log.Logger that is meant to be used in the ErrorLog field of an http.Server to log HTTP server errors."